Products/Zuul/routers/zep.py

```diff
@@ -390,6 +390,10 @@ def queryGenerator(self, sort='lastTime', dir='desc', evids=None, excludeIds=Non
         @rtype:   generator
         @return:  Generator returning events.
         """
+
+        if isinstance(params, basestring):
+            params = loads(params)
+
```

Comment:
if this code is being hit by the "export" button in the UI, params will be a json string, so parse it into a dict